src/praisonai/praisonai/integrations/db_session_adapter.py

```
@@ -37,16 +37,43 @@ def __init__(self, db: Any):
+        if conv is not None:
+            return conv
+        orchestrator = getattr(self._db, "_orchestrator", None)
+        return getattr(orchestrator, "conversation", None) if orchestrator else None
+
+    def _purge_persisted_messages(self, session_id: str) -> None:
+        """Remove persisted messages when the session is cleared or deleted."""
+        conv = self._conversation_store()
+        if conv is not None and hasattr(conv, "delete_messages"):
+            try:
+                conv.delete_messages(session_id, None)
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)